workflows/reference_files/2.3/reference_files.smk

```diff
@@ -51,14 +53,16 @@ rule create_star_index:
         "genomes/{genome_build}/star_index/star-{star_version}/gencode-{gencode_release}/overhang-{star_overhang}.log"
     conda: CONDA_ENVS["star"]
     threads: 12
+    resources:
+        mem_mb = 42000
     shell:
         op.as_one_line("""
         mkdir -p {output.index}
             &&
         STAR --runThreadN {threads} --runMode genomeGenerate --genomeDir {output.index}
         --genomeFastaFiles {input.fasta} --sjdbOverhang {wildcards.star_overhang}
-        --sjdbGTFfile {input.gtf} --outTmpDir {output.index}/_STARtmp > {log} 2>&1
```

Comment:
Can someone please confirm this was a bug?

Reply:
I don't think it is a bug. That should redirect both STDOUT and STDERR to {log}. I can confirm at least that stdout is redirected, since I didn't have any errors when I generated the files I can't confirm re: STDERR. But I think you were able to see errors in this file when you were having memory issues on Cedar, right?

Reply:
Anyway, moving it to the end of the STAR command is fine.